generators/client/templates/angular/src/main/webapp/app/shared/auth/_user-route-access-service.ts

```diff
@@ -16,6 +16,11 @@ export class UserRouteAccessService implements CanActivate {
     }
 
     canActivate(route: ActivatedRouteSnapshot, state: RouterStateSnapshot): boolean | Promise<boolean> {
+
+        if (route.data['authorities'].length == 0) {
```

Comment:
we definitely could have!
`route.data['authorities'] && route.data['authorities'].length == 0`